SECURITY.md

```diff
@@ -0,0 +1,45 @@
+#Gratipay Responsible Disclosure Policy 
+
+We take the security of our systems seriously, and we value the security community. The responsible disclosure of security vulnerabilities helps us ensure the security and privacy of our users.
+
+**Guidelines**  
+
+We require that all researchers:
```

Comment:
"Require" seems like an overreach. What are we going to do if someone doesn't follow these guidelines? I suggest we change "require" to "ask" instead.